postgres/queries/organisations.sql

```diff
@@ -82,3 +87,15 @@ FROM
     organisations
 WHERE
     id = $1;
+
+-- name: GetMemberByEmail :one
```

Comment:
ik weet niet of we afdwingen of een email uniek moet zijn in een organisatie, maar dat lijkt me wel zo handig

Reply:
Een email moet globaal uniek zijn

Reply:
Inderdaad, e-mail is al globaal uniek dus per definitie ook binnen een organisatie. Je mag daar ook gerust van uitgaan, dat is niet iets dat ik ooit zie veranderen (aangezien e-mailadressen in de echte wereld ook altijd uniek per gebruiker zullen zijn)

Reply:
ok, dan mag ik die LIMIT idd blijkbaar weglaten, daar ging het me vooral om :-)